app/assets/javascripts/components/flexible_roster.jsx

```diff
@@ -104,7 +104,7 @@ export default React.createClass({
       <tr className='column-groups'>
         {superHeaders.map((superHeader, index) => {
           return (
-            <th key={index} className={superHeader.label == null ? '' : 'column-group'} colSpan={superHeader.span}>
+            <th key={index} className={superHeader.label === null ? '' : 'column-group'} colSpan={superHeader.span}>
```

Comment:
We might want `undefined` to have the same behavior here (as it would with double equals)?  Could flip this to be "if not a string" if that's what we really mean.